src/jquery.multiselect.js

```diff
@@ -284,13 +292,13 @@
 
     /**
      * Processes option and optgroup tags from underlying select to construct the menu's option list
-     * This clears the items currently in $checkboxContainer
+     * This clears the items currently in $checkboxes
      * Defers to _makeOption to actually build the options
      * Resets the input ID counter
      * @param {object} $element element whose option/group tags need to be converted
-     * @param {object} $checkboxContainer widget's container to append the built options to
+     * @param {object} $checkboxes widget's container to append the built options to
      */
-    _buildOptionList: function($element, $checkboxContainer) {
+    _buildOptionList: function($element, $checkboxes) {
```

Comment:
Now that the recursion is gone, we could probably use the instance variables of the widget instead of passing these parameters in. Don't need to fix it in this PR, just something I wanted to make a note of.

Reply:
Agreed.